enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md

```diff
@@ -0,0 +1,378 @@
+---
+title: share-secrets-and-configmaps-across-namespaces-via-a-csi-driver
+
+authors:
+  - "@gabemontero"
+
+reviewers:
+  - "@bparees"
+  - "@adambkaplan"
+  - "@derekwaynecarr"
+  - "@gnufied"
+  
+approvers:
+  - "@bparees"
+  - "@adambkaplan"
+  - "@derekwaynecarr"
+  - "@gnufied"
+  
+creation-date: 2020-03-17
+
+last-updated: 2020-03-30
+
+<!-- status: provisional|implementable|implemented|deferred|rejected|withdrawn|replaced -->
+status: provisional
+
+
+  
+---
+
+# Share Secrets And ConfigMaps Across Namespaces via a CSI Driver
+
+
+## Release Signoff Checklist
+
+- [/] Enhancement is `implementable`
+- [/] Design details are appropriately documented from clear requirements
+- [/] Test plan is defined
+- [/] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions [optional]
+
+> 1. Is a CSI `SecretProviderClass` based implementation recommended?  Received some varying guidance.  Also, per PR
+>review from @jsafrane, of which I can only concur based on my examination, the example at https://github.com/kubernetes-sigs/secrets-store-csi-driver
+>does not appear to handle refreshing of the data after initial provisioning.  So it does not appear to be something we
+>can use as is.  But do we want to take on adding such a feature upstream to that repo? **This question most
+>likely can be deferred until implementation.**
+> 2. What will be the precise means of interacting with the host filesystem.  The hostpath plugin actually uses the 
+>simple golang `os.MkdirAll` for example when creating mount access based volumes.  Can we do that in all our supported
+>scenarios?  We've also received suggestions around tmpfs or read only bind-mounts of a directory from the CSI driver.
+>**This question most likely can be deferred until implementation.**
+> 3. SELinux implications will need to be properly understood.  Minimally, a consuming Pod needs the correct SELinux 
```

Comment:
not sure I see how this can be deferred. You can get some ways down the road, but I don't think you can deliver without having selinux working with restricted SCC

Reply:
Yeah I was trying to distinguish between merging this PR vs. actual delivery of the feature in OCP.

I'll update to make sure that is clearer.

thanks